src/components/calcite-flow/calcite-flow.tsx

```diff
@@ -53,6 +58,18 @@ export class CalciteFlow {
     this.flowItemObserver.disconnect();
   }
 
+  // --------------------------------------------------------------------------
+  //
+  //  Events
+  //
+  // --------------------------------------------------------------------------
+
+  @Listen("calciteFlowItemBackClick") handleCalciteFlowItemBackClick() {
+    if (this.handleBack) {
```

Comment:
fixme: Should be like this: https://github.com/Esri/calcite-components/blob/master/src/components/calcite-modal/calcite-modal.tsx#L185